### ISSUE TITLE
[2021]Convolution-Free Medical Image Segmentation using Transformers

### ISSUE THREAD

**Issue by yusukekyokawa** [yusukekyokawa/paper_list#85]
### 論文リンク
https://arxiv.org/abs/2102.13645v1
### 著者/所属機関
Davood Karimi, Serge Vasylechko, Ali Gholipour
### conf/journal
arxiv
### year
2021
## どんな論文か？
Like other applications in computer vision, medical image segmentation has been most successfully addressed using deep learning models that rely on the convolution operation as their main building block. Convolutions enjoy important properties such as sparse interactions, weight sharing, and translation equivariance. These properties give convolutional neural networks (CNNs) a strong and useful inductive bias for vision tasks. In this work we show that a different method, based entirely on self-attention between neighboring image patches and without any convolution operations, can achieve competitive or better results. Given a 3D image block, our network divides it into n3 3D patches, where n=3 or 5 and computes a 1D embedding for each patch. The network predicts the segmentation map for the center patch of the block based on the self-attention between these patch embeddings. We show that the proposed model can achieve segmentation accuracies that are better than the state of the art CNNs on three datasets. We also propose methods for pre-training this model on large corpora of unlabeled images. Our experiments show that with pre-training the advantage of our proposed network over CNNs can be significant when labeled training data is small.
## 新規性

ここがすごい
## 手法
![image](https://user-images.githubusercontent.com/39184448/109641285-e1e21b00-7b94-11eb-9aa8-50cf4090a402.png)

**Comment by yusukekyokawa** [comment]
これ面白そうだね！
